app/models/version.rb

```diff
@@ -28,7 +28,9 @@ def self.previous_production
   end
 
   def self.current_preview
-    Version.preview.newest.first
+    return nil if current_production && preview.newest.first.created_at < current_production.created_at
+
+    preview.newest.first
```

Comment:
Are there plans to clean up `versions`?  Just wondering if there will ever be a case where there are no preview `versions`.
I think a safe way to write this would be:
```ruby
  preview.newest.where('created_at > ?', current_production.created_at).first
```
Can there ever not be a `current_production`?

Reply:
Upon building the db, we will always have one preview version and one current production `version`. So I don't think we will ever have a case where there are no preview or production `versions`.

I've put in your suggested change and tested it. It looks good to me.

Reply:
Got it.  So then I thought, "why does seeding the data create two `versions`?".
I looked at `02_data.rb` and it is acting in the old style:  It creates a preview version.  Then it creates a production version with the same version number.
Can we change that so that it is consistent with this new paradigm and just updates the preview `version` to production?

Reply:
Then this method definition would work with something like this:
```ruby
    cp = preview.newest
    cp = cp.where('created_at > ?', current_production.created_at) if current_production
    cp.first
```

Reply:
That makes sense. Changes are in.